lib/figures/pawn.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative 'figure'
+
+class Pawn
+  attr_reader :row, :column, :possible_moves, :color, :display
+
+  include Figure
+
+  def initialize(color, row, column)
+    @color = color
+    @row = row
+    @column = column
+    @display = @display = { white: '♟', black: '♙' }
+    @move_pattern = find_move_pattern
+    @possible_moves = find_possible_moves
+  end
+
+  def find_move_pattern
```

Comment:
ah, so move pattern is dynamic! That should be a method for all figures then. Special cases are bad, generalize what you can : )